packages/grid/_modules_/grid/utils/keyboardUtils.ts

```diff
@@ -1,10 +1,10 @@
 export const MULTIPLE_SELECTION_KEYS = ['Meta', 'Control'];
-export const isMultipleKey = (code: string): boolean => MULTIPLE_SELECTION_KEYS.indexOf(code) > -1;
-export const isTabKey = (code: string): boolean => code === 'Tab';
-export const isSpaceKey = (code: string): boolean => code === 'Space';
-export const isArrowKeys = (code: string): boolean => code.indexOf('Arrow') === 0;
-export const isHomeOrEndKeys = (code: string): boolean => code === 'Home' || code === 'End';
-export const isPageKeys = (code: string): boolean => code.indexOf('Page') === 0;
+export const isMultipleKey = (key: string): boolean => MULTIPLE_SELECTION_KEYS.indexOf(key) > -1;
```

Comment:
Didn't @oliviertassinari rename those to `key` not that long ago? I personally am ok with either one but `key` does sound more appropriate as it represents the actual key name, the `code` is different.

Reply:
@DanailH The change looks great, what's your concern?

Reply:
Actually, why is it implemented like this? Why didn't we use a stateless approach?

https://github.com/mui-org/material-ui/blob/67e75ced891966ef38f9fff3253db8582d0fbe87/packages/material-ui-lab/src/TreeView/TreeView.js#L645

Reply:
Ignore/resolve my comment. For some reason, I thought that the change is from `key` to `code`.

Reply:
Yes we could refactor isMultipleKey..., it will impact sorting, selection...
ATM I didn't feel it was too much a pain so left it

Reply:
Agree, it's definitely off-topic for the bug we are solving :)